src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/_help.py

```diff
@@ -280,3 +280,21 @@
     type: command
     short-summary: Lists the firewall rules.
     """
+helps['sql server vnet-rule'] = """
+    type: group
+    short-summary: Manage a server's virtual network rules.
+    """
+helps['sql server vnet-rule update'] = """
+    type: command
+    short-summary: Updates a virtual network rule.
+    """
+helps['sql server vnet-rule create'] = """
+    type: command
+    short-summary: Creates a virtual network rule which allows access to an Azure Sql server.
+
+    examples:
+        - name: Create a vnet rule by providing the subnet id. Vnet name is not required.
+          text: az sql server vnet-rule create --subnet /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rgName/providers/Microsoft.Network/virtualNetworks/vnetName/subnets/subnetName
+        - name: Create a vnet rule by providing the vnet and subnet name. The subnet id is created by taking the resource group name and subscription id of the sql server.
+          text: az sql server vnet-rule create --subnet subnetName --vnet-name vnetName
```

Comment:
I should have thought of this earlier, but it's not typical to have `-name` in a param name. So I think this param should probably be named `--vnet` instead. Anyone else has opinions on this?

Reply:
In this case, vnet-name is the correct name.  "Subnet" accepts a name or ID.  Vnet name is only required when the subnet is not an ID, and that's why it is called vnet-name.  If vnet could be a name or ID, then it would just be called "vnet".

Reply:
So you either provide:
`--subnet ID` or `--subnet NAME --vnet-name NAME`

Reply:
Ok sounds good if that's the standard.